src/site/content/en/blog/bluetooth/index.md

```diff
@@ -558,7 +556,6 @@ Hero image by [SparkFun Electronics from Boulder, USA].
 [Promises tutorial]: /promises
 [Arrow functions]: https://developer.mozilla.org/docs/Web/JavaScript/Reference/Functions/Arrow_functions
 [Bluetooth GATT Battery Service]: https://www.bluetooth.com/specifications/gatt/
-[the standardized Bluetooth GATT services]: https://www.bluetooth.com/specifications/assigned-numbers/
```

Comment:
It may still be worth linking to this when explaining the examples using a standardized service as the source of the magic numbers.

Reply:
I'm already linking to https://www.bluetooth.com/specifications/specs/ for standardized services and characteristics. See the "standardized Battery Level Characteristic" link at https://deploy-preview-5969--web-dev-staging.netlify.app/bluetooth/#read for instance

Reply:
If this is useful in this context, why add extra layers of searching and linking?

Reply:
@jpmedley https://www.bluetooth.com/specifications/specs/ seems better than https://www.bluetooth.com/specifications/assigned-numbers/ when talking about specific standardized services and characteristics 

In other words, https://www.bluetooth.com/specifications/specs/#:~:text=Battery%20Service seems easier to find than going to https://www.bluetooth.com/specifications/assigned-numbers/, click "16-bit UUIDs", and find "Battery Service".

Reply:
>In other words, https://www.bluetooth.com/specifications/specs/#:~:text=Battery%20Service seems easier to find than going to https://www.bluetooth.com/specifications/assigned-numbers/, click "16-bit UUIDs", and find "Battery Service".

I'm not disputing that. What I'm saying is that by removing any kind of link from this text you're replacing one scroll with another. Replace the URL, but don't remove the link.

Reply:
I've removing the link because it doesn't make sense anymore to link to it as we don't handle human-readable names for GATT services, characteristics, and descriptors.

```diff
- If your Bluetooth GATT Service is not on the list of [the standardized Bluetooth
- GATT services] though, you may provide either the full Bluetooth UUID or a short
- 16- or 32-bit form.
+ You can use either the full Bluetooth UUID or a short 16- or 32-bit form for Bluetooth
+ GATT Services.
```